kube-tail

```diff
@@ -1,84 +1,6 @@
-#!/bin/bash
-#
+#!/bin/sh
 
-set -eou pipefail
+SCRIPT_DIR=$( cd "$(dirname "$0")" ; pwd -P )
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2164
```
$ shellcheck kube-tail

In kube-tail line 3:
SCRIPT_DIR=$( cd "$(dirname "$0")" ; pwd -P )
              ^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```